src/services/registrate.js

```diff
@@ -0,0 +1,20 @@
+const ENDPOINT = `${import.meta.env.VITE_SERVER_URL}/api/v1/users`
```

Comment:
i would move this constants to a setup file which includes all the env variables that you have